appsensor-core/src/main/java/org/owasp/appsensor/core/IAppsensorEntity.java

```diff
@@ -0,0 +1,14 @@
+package org.owasp.appsensor.core;
+
+import java.io.Serializable;
+
+/**
+ * Interface denoting methods required to be provided by appsensor entity objects
+ * @author Maik Jäkel(m.jaekel@xsite.de) http://www.xsite.de
+ */
+public interface IAppsensorEntity extends Serializable{
```

Comment:
@MaikJaek What was the reason for needing this interface? You just think it's a better design or was this for something more?